FirebaseRemoteConfig/Sources/FIRRemoteConfigComponent.m

```diff
@@ -23,6 +23,17 @@
 #import "Interop/Analytics/Public/FIRAnalyticsInterop.h"
 
 @implementation FIRRemoteConfigComponent
+static FIRRemoteConfigComponent *_sharedDefaultAppRemoteConfigSingleton = nil;
+
++ (FIRRemoteConfigComponent *)sharedDefaultAppRemoteConfigSingletonWithDefaultApp:(FIRApp *)app {
```

Comment:
We probably need some strategy to create a singleton for the default Provider and Interop. They will return the same object.

Reply:
Checked RC Component from Crashlytics and FirePerf, confirm they have the same the same RC Component: https://screenshot.googleplex.com/9NXgjPbmtxoSrSe